azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/BottomBarView.kt

```diff
@@ -53,6 +53,7 @@ internal fun BottomBarView(
 }
 
 private fun sendButtonOnclick(postAction: (Action) -> Unit, messageInputTextState: MutableState<String>) {
+    if (messageInputTextState.value.isBlank()) return
```

Comment:
can you please check with Alex, if we need to show grey send message button when disabled?

Reply:
Should have disable state.